packages/test/test-utils/src/testFluidObject.ts

```diff
@@ -200,20 +199,24 @@ export class TestFluidObjectFactory implements IFluidDataStoreFactory {
 			},
 		);
 
-		return new runtimeClass(
+		const runtime = new runtimeClass(context, dataTypes, existing, async () => {
+			await instance.initialize(true);
+			return instance;
+		});
+
+		const instance: TestFluidObject = new TestFluidObject(
+			runtime,
+			// This works because 'runtime' is an instance of runtimeClass (which is a FluidDataStoreRuntime and
+			// thus implements IFluidDataStoreChannel) which passes itself as the parameter to this function.
+			runtime,
```

Comment:
Get rid of the comment in the middle, or rewrite it? It made sense when this was inside the entryPoint initializer; now I think it might confuse people reading it.

Reply:
I suppose that's true, it's a little weird either way.  I'll remove it and just annotate the params.